src/server/active_tran_server.cpp

```diff
@@ -183,21 +185,25 @@ active_tran_server::init_page_server_hosts (const char *db_name)
       const int local_exit_code = connect_to_page_server (node, db_name);
```

Comment:
Keep all connections by converting m_page_server_conn into a vector of connections. Add all successful connections to the vector.

When requests are pushed, use the first connection in the vector.

Reply:
Done